packages/ove-asset-manager/src/OVE.Service.AssetManager/Controllers/HomeController.cs

```diff
@@ -3,25 +3,23 @@
 using Microsoft.Extensions.Logging;
 using OVE.Service.AssetManager.Models;
 
-namespace OVE.Service.AssetManager.Controllers
-{
-    public class HomeController : Controller
-    {
+namespace OVE.Service.AssetManager.Controllers {
+    public class HomeController : Controller {
         private readonly ILogger<HomeController> _logger;
 
         public HomeController(ILogger<HomeController> logger) {
             _logger = logger;
         }
-        public IActionResult Index()
-        {
-            _logger.Log(LogLevel.Critical,"Read Index page");
+
+        public IActionResult Index() {
             return View();
         }
 
         [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
-        public IActionResult Error()
-        {
-            return View(new ErrorViewModel { RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier });
+        public IActionResult Error() {
+            var errorViewModel = new ErrorViewModel {RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier};
+            _logger.LogError("Viewing Error"+errorViewModel);

```

Comment:
What's the purpose of logging a string representation of the ErrorViewModel instance?

Reply:
So that full details of the error are placed into the common logging system.

Reply:
Maybe we want to log the ` HttpContext.Trace` rather than the ` HttpContext.TraceIdentifier`?

Reply:
The TraceId enables developers to look up the events in the TraceLog on the server if Tracing is enabled (off by default in prod, on by default in dev). The actual Trace is not accessible as far as I know from code (e.g. `HttpContext.Trace` doesn't exist).
I have logged the Request details in the most recent comment which should suffice for a level of detail to enable debugging.

Reply:
Tested and this works.

```
fail: OVE.Service.AssetManager.Controllers.HomeController[0]
ovehub-ove-asset-manager_1_75c96d4f2005 |       Viewing Error: OVE.Service.AssetManager.Models.ErrorViewModel Request: Microsoft.AspNetCore.Http.Internal.DefaultHttpRequest
ovehub-ove-asset-manager_1_75c96d4f2005 | fail: Microsoft.EntityFrameworkCore.Database.Command[20102]
ovehub-ove-asset-manager_1_75c96d4f2005 |       Failed
```